libs/docs/core/table/examples/table-focusable-example/table-focusable-example.component.html

```diff
@@ -10,11 +10,11 @@
     </thead>
     <tbody fd-table-body>
         <tr fd-table-row [focusable]="true" *ngFor="let item of [0, 1, 2, 3, 4, 5]">
-            <td fd-table-cell>Whole</td>
-            <td fd-table-cell>Row</td>
-            <td fd-table-cell>Can also</td>
-            <td fd-table-cell>Be</td>
-            <td fd-table-cell>Focused</td>
+            <td fd-table-cell [focusable]="true">Whole</td>
+            <td fd-table-cell [focusable]="true">Row</td>
+            <td fd-table-cell [focusable]="true">Can also</td>
+            <td fd-table-cell [focusable]="true">Be</td>
+            <td fd-table-cell [focusable]="true">Focused</td>
```

Comment:
Just wondering, any reason for this instead of using the allCellsFocusable input on the table?